Bottom: a53cb873e5e5b1d8dc70af739ecfd2d547efb5c2
Top:    258d40f58464141588bacd37138434069c69c9ab
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-10-27 16:53:30 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index af5aa3e..70cc17f 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
